database/factories/ConsumableFactory.php

```diff
@@ -116,4 +117,16 @@ public function requiringAcceptance()
             $consumable->category->update(['require_acceptance' => 1]);
         });
     }
+
+    public function checkedOutToUser(User $user = null)
+    {
+        return $this->afterCreating(function (Consumable $consumable) use ($user) {
+            $consumable->users()->attach($consumable->id, [
+                'consumable_id' => $consumable->id,
+                'created_at' => Carbon::now(),
+                'user_id' => 1,
```

Comment:
We should avoid hard-coding this and use a factory instead. I think the following should do it.

```suggestion
                'user_id' => User::factory(),
```